scripts/build.sh

```diff
@@ -61,6 +65,15 @@ ubuntu_setup () {
     else
         echo "Configuring static Ubuntu build"
     fi
+
+    if [ $ARCH = "aarch64" ]
+    then
+	echo "Detected ARM64 Linux!"
+	# TODO: We should start using container builds in actions!
```

Comment:
What are "container builds"?

Reply:
Just makes the host invoke a [docker container](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container).